<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1091</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Задание №1</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>611</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Save">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>260</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Open">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Help">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>380</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Справка</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>931</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;Ваш первый проект на данном курсе - текстовый редактор. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;Добавьте на форму виджет текстового редактора. Добавьте две кнопки: сохранить и открыть. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;Предоставьте возможность пользователю выбрать куда сохранять файл и какой файл открывать. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;Добавьте в текстовый редактор справку. Текстовый файл с описанием разместить в ресурсах программы.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Задание №2</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>951</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;В этом задании нужно создать парсер Авито (будем рассматривать только Москву). &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;Пользователь должен иметь возможность вводить запрос (то, что он хочет найти). &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;После этого с сайта авито с первой страницы нужно получить следующую информацию: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;название объявления, ссылку на объявление, цена, метро (если есть), сколько метров от метро (если есть). &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Helvetica','sans-serif'; font-size:10pt; color:#2c2d30; background-color:#ffffff;&quot;&gt;Эту информацию нужно отобразить в виде таблицы.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>821</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Find">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>190</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>641</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите запрос:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
